tajo-client/src/main/proto/ClientProtos.proto

```diff
@@ -198,6 +198,7 @@ message CreateTableRequest {
   required TableProto meta = 4;
   required string path = 5;
   optional PartitionMethodProto partition = 6;
+  required bool hasSelfDescSchema = 7;
```

Comment:
In this message scheme, SchemaProto is required. it may have an empty Schema? This semantic seems to be different from TableDesc. It's trivial thing. If we see more consistent semantic, the code would be more comprehensive.